### DISCUSSION TITLE
what the picture mean? use version 3.5.5-p1

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#811] [Q&A]
![image](https://user-images.githubusercontent.com/50607317/121625844-db721200-caa6-11eb-9803-24afdd285644.png)
@junoyoon 
Do I need to deal with the content of log output during script verification？
@donggyu04

**Comment by donggyu04** [comment]
@zhougy1989 

at some point, debug logs are stamped in the controller validation log. which is not intended.
but, the validation log is not large and can be considered for debugging purposes. Ignore if you don't need it.
